src/hooks/useUpdatePackageFilesMutation.ts

```diff
@@ -58,6 +58,28 @@ export function useUpdatePackageFilesMutation({
           }
         }
       }
+      for (const initialFile of initialFilesLoad) {
```

Comment:
```suggestion
      for (const initialFile of initiallyLoadedFiles) {
```

Reply:
variables should be nouns, not verbs. Functions can/should be verbs or questions